experimental/embeddings/static/flair.py

```diff
@@ -0,0 +1,180 @@
+import logging
+from pathlib import Path
+from pickle import UnpicklingError
+from typing import Optional
+
+import flair
+import gensim
+import numpy as np
+import torch
+from flair.embeddings import WordEmbeddings
+from flair.file_utils import cached_path
+from torch import nn
+
+logging.getLogger("flair").setLevel(logging.INFO)
+
+
+class WordEmbeddingsPL(WordEmbeddings):
+    """Standard static word embeddings, such as GloVe or FastText for polish language."""
```

Comment:
I would fix this docstring and add info that it is borrowed from flair and that fixes error with embedding loading